### PR TITLE
added bfarm icd10gm

### PR DESCRIPTION
**Pull request by enola-dkfz**

### CODE REVIEW

**Comment by enola-dkfz** [comment]
Broke the test, no time to fix now

**Review by patrickskowronekdkfz — APPROVED** [review]

File: src/util.rs [hunk 1]
```diff
@@ -433,72 +434,24 @@ mod test {
         assert!(!is_cql_tampered_with(decoded_library));
     }
 
+    #[test]
+    fn test_replace_cql_all() {
+        for (key, value) in REPLACE_MAP.iter() {
+            let decoded_library: &str = key;
+            let expected_result: &str = value;
```
Comment (Threated) [inline src/util.rs:441]: I dont think you need these vars just rename them when destructing in the loop

**Review by Threated — APPROVED** [review]

File: src/util.rs (hunk 1 above)
Comment (Threated) [reply src/util.rs:441]: Really even if you do .as_ref() or something like that?

Comment (enola-dkfz) [reply src/util.rs:441]: where do I put .as_ref()?

Comment (Threated) [reply src/util.rs:441]: In the assert but it's not too important if you want you can merge

Comment (enola-dkfz) [reply src/util.rs:441]: this works: pretty_assertions::assert_eq!(replace_cql(*decoded_library).as_str(), *expected_result);

**Pull request merged**